templates/components/common/footer.html

```diff
@@ -8,20 +8,26 @@
 <footer class="footer" role="contentinfo">
     <div class="container">
         <section class="footer-info">
-            <article class="footer-info-col footer-info-col--small" data-section-type="footer-webPages">
-                <h5 class="footer-info-heading">{{lang 'footer.navigate'}}</h5>
-                <ul class="footer-info-list">
-                    {{#each pages}}
-                    <li>
-                        <a href="{{url}}">{{name}}</a>
-                    </li>
-                    {{/each}}
-                    <li>
-                        <a href="{{urls.sitemap}}">{{lang 'sitemap.page_title'}}</a>
-                    </li>
-                </ul>
-            </article>
-
+            {{#if theme_settings.social_icon_placement_bottom '===' 'bottom_left'}}
+                <article class="footer-info-col footer-info-col" data-section-type="footer-webPages">
+                    <h5 class="footer-info-heading">{{lang 'social.connect'}}</h5>
+                    {{> components/common/social-links}}
+            {{else}}
+                <article class="footer-info-col footer-info-col--small" data-section-type="footer-webPages">
+            {{/if}}
```

Comment:
it's supposed to be in addition to the social connect h5.  so there can be two h5's:

![screen shot 2016-12-07 at 1 03 29 pm](https://cloud.githubusercontent.com/assets/1357197/20986448/9dfca66a-bc7d-11e6-91c6-40b6e7725b17.png)

here 'Connect With Us' is the first and 'Navigate' the second.  i moved the conditional to the article's class instead of conditionally showing the article itself, hoping that it's clearer that way.